docs/user/document/publisher/contact_details-usage.en.md

```diff
@@ -1 +1,9 @@
 # Contact Details - Usage
+
+Lists means of contacting the publisher of the Advisories.
```

Comment:
Wrong capitalization of advisories; means => options

Reply:
updated